nixos/modules/services/security/pbis.nix

```diff
@@ -0,0 +1,148 @@
+{ config, lib, pkgs, ... }:
```

Comment:
I would encourage you to add yourself as a maintainer to this module, if you are interested in doing so.